src/main/java/CST438hw2/service/CityService.java

```diff
@@ -0,0 +1,41 @@
+package CST438hw2.service;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import CST438hw2.domain.*;
+
+@Service
+public class CityService {
+    @Autowired
+    private CityRepository cityRepository;
+    @Autowired
+    private CountryRepository countryRepository;
+    @Autowired
+    private WeatherService weatherService;
+    
+    public CityInfo getCityInfo(String cityName) { 
+        List<City> cities = cityRepository.findByName(cityName);
+        if ( cities.size()==0) {
+            return null;
+            
+        } else {
+            City city= cities.get(0);
+            System.out.println("**********************");
+            System.out.println(city.getCountry().getCode());
```

Comment:
More print statements that appear to be for debug purposes. Should also be removed before final commit